test/test_pebble.py

```diff
@@ -46,7 +46,7 @@ def datetime_nzdt(y: int, m: int, d: int, hour: int, min: int, sec: int, micro:
     return datetime.datetime(y, m, d, hour, min, sec, micro, tzinfo=tz)
 
 
-class TestTypes(unittest.TestCase):
+class TestTypes:
```

Comment:
`maxDiff` below appears unused.

There seem to be a bunch of these in this file.

Now that we're not using plain unittest, I think it's safe to remove these...

Reply:
Thanks, there are a couple of these in other files, too. I will delete them once and for all in this PR.

Reply:
If we need this functionality (huge diffs), then I think that the pytest way would be to stash some flag into `pyproject.toml`.
On the other hand, pytest diffs are better, so maybe we don't need huge diffs. I'd check by breaking a couple tests to see the actual diffs.